pysatModels/models/ucar_tiegcm.py

```diff
@@ -35,13 +41,13 @@
 tags = {'': 'UCAR TIE-GCM file'}
 inst_ids = {'': ['']}
 
-# specify using xarray (not using pandas)
+# Specify using xarray (not using pandas)
 pandas_format = False
 
 # ----------------------------------------------------------------------------
 # Instrument test attributes
 
-_test_dates = {'': {'': dt.datetime(2019, 1, 1)}}
+_test_dates = {'': {'': dt.datetime(2020, 1, 10)}}
 _test_download = {'': {'': False}}
```

Comment:
```suggestion
_test_download_ci = {'': {'': False}}
```
This should fix the tests.  Instruments with download support that are skipped in the CI environment should use this flag, otherwise pytest will check for a warning message that downloads are not supported.